### DISCUSSION TITLE
Welcome to accessibility-resources Discussions!

### DISCUSSION THREAD

**Discussion by sarahesbie** [codelikeagirlau/accessibility-resources#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by laurenchamps** [comment]
I'm Lauren (she/her), and I live on Kaurna land (Adelaide). 

I'm a newbie, started with the CLG Foundations of Web Dev course and have since been doing self-directed online study (The Odin Project Foundations, CS50: Intro to CS and currently working through The Odin Project Full Stack Javascript path) and am hoping to work as a developer in the future. I'm familiar with HTML, CSS, JavaScript and a bit of python and SQLite. I've used git and github for my own repositories but never collaborated with others so that'll be good to learn! 

I'm very keen to embed accessibility into my work from the get-go rather than as an afterthought so I'm hoping I'll learn a lot from contributing to this project and also hopefully we'll have a great resource I can keep referring to! 

Thanks for setting this up - I appreciate the opportunity to have a go at open source with a bunch of friendly folk like yourselves 😄

**Comment by jazzicoda** [comment]
Hi everyone! It would be fun to have a downloadable CSS cheatsheet as an asset for the website. Something like the CSS-TRICKS CSS Grid and Flexbox posters: 
https://css-tricks.com/snippets/css/complete-guide-grid/ 
https://css-tricks.com/snippets/css/a-guide-to-flexbox/
Fun fact - CSS-TRICKS is "Powered by Digital Ocean" - sponsors of Hacktoberfest!

**Reply by jazzicoda** [reply]
Another random thought.... ABILITY is within Accessibility.... **A**_ccessi_**BILITY**
noun
1. possession of the means or skill to do something.
2. talent, skill, or proficiency

A new platform to give devs the ABILITY to make websites accessible.... just random thought which others could build on....
